alica_engine/include/engine/scheduler/Scheduler.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "engine/scheduler/Job.h"
+#include <vector>
+
+namespace alica
+{
+namespace scheduler
+{
+    class Scheduler
```

Comment:
The constructor should take the number of threads that will run inside the scheduler.